src/inmanta/data/model.py

```diff
@@ -725,7 +725,7 @@ class DiscoveredResource(BaseModel):
     """
 
     discovered_resource_id: ResourceIdStr
-    values: JsonType
+    values: dict[str, object]
```

Comment:
I would expect JsonType to be more correct no? 

(at least, it should be?)

Reply:
See discussion of last week